packages/services/src/epcisEvents/objectEvent.ts

```diff
@@ -1,33 +1,34 @@
 import { VerifiableCredential } from '@vckit/core-types';
 import { IService } from '../types/index.js';
-import { issueVC } from '../vckit.service.js';
 import { IContext } from './types.js';
 import { getIdentifierByObjectKeyPaths } from './helpers.js';
-import { uploadJson } from '../storage.service.js';
 import { generateUUID } from '../utils/helpers.js';
-import {
-  ILinkResolver,
-  ILinkResponse,
-  IdentificationKeyType,
-  LinkType,
-  MimeType,
-  createLinkResolver,
-} from '../linkResolver.service.js';
 
+import { uploadJson } from '../storage.service.js';
+import { issueVC } from '../vckit.service.js';
+import { registerLinkResolver } from '../linkResolver.service.js';
+
+/**
+ * Process object event, issue VC, upload to storage and register link resolver
+ * @param data - object event data, e.g. { data: { herd: { NLIS: 'NH020188LEJ00012' } } }
+ * @param context - object event context
+ * @returns
+ */
 export const processObjectEvent: IService = async (data: any, context: IContext): Promise<any> => {
   try {
     const vckitContext = context.vckit;
```

Comment:
For the context parameter, I think we should have a validate function per event that will validate the required data to process the event